docs/reference/getting-started.asciidoc

```diff
@@ -95,7 +95,7 @@ Replication is important for two primary reasons:
 To summarize, each index can be split into multiple shards. An index can also be replicated zero (meaning no replicas) or more times. Once replicated, each index will have primary shards (the original shards that were replicated from) and replica shards (the copies of the primary shards).
 The number of shards and replicas can be defined per index at the time the index is created. After the index is created, you may change the number of replicas dynamically anytime but you cannot change the number of shards after-the-fact.
 
-By default, each index in Elasticsearch is allocated 5 primary shards and 1 replica which means that if you have at least two nodes in your cluster, your index will have 5 primary shards and another 5 replica shards (1 complete replica) for a total of 10 shards per index.
+By default, each index in Elasticsearch is allocated 5 primary shards and 1 replica which means that you need to have at least two nodes in your cluster(since replica is not stored on the same node and we have 1 replica here), your index will have 5 primary shards and another 5 replica shards (1 complete replica) for a total of 10 shards per index.
 
```

Comment:
This subtly changes the meaning--the intent was to describe a possible scenario. Instead, I would suggest splitting off the scenario into a separate sentence. 

By default, each index in Elasticsearch is allocated 5 primary shards and 1 replica. If you have two nodes in your cluster, your index will have a total of 10 shards: 5 primary shards on one node and 5 replica shards (1 complete replica) on the other.  Replicas are not stored on the same node as an index's primary shards.